src/pyseekdb/client/client_seekdb_embedded.py

```
@@ -278,34 +278,38 @@ def create_database(self, name: str, tenant: str = DEFAULT_TENANT) -> None:
+
     def get_database(self, name: str, tenant: str = DEFAULT_TENANT) -> Database:
         """
         Get database object (tenant parameter ignored for embedded mode)
-        
+
         Args:
             name: database name
             tenant: ignored for embedded mode (no tenant concept)
         """
         logger.info(f"Getting database: {name}")
         sql = f"SELECT SCHEMA_NAME, DEFAULT_CHARACTER_SET_NAME, DEFAULT_COLLATION_NAME FROM information_schema.SCHEMATA WHERE SCHEMA_NAME = '{name}'"
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)